next-app/src/db/db.json

```diff
@@ -8,6 +112,9 @@
       "region": "Canada",
       "type": "Rye",
       "abv": 45,
+      "description": "This one changed the way we think of bourbon, all because one man changed the way he thought about making it. Bill Samuels, Sr., simply wanted a whisky he would enjoy drinking. Never bitter or sharp, Maker's Mark® is made with soft red winter wheat, instead of the usual rye, for a one-of-a-kind, full-flavored bourbon. To ensure consistency, we rotate every barrel by hand and age our bourbon to taste, not time. Each and every bottle of Maker's® is still hand-dipped in our signature red wax at our distillery in Loretto, KY., just like Bill, Sr. would have wanted.",
```

Comment:
Please add either description for every whiskey, or delete this from the commit, to ensure consistency on the main branch.